packages/server/src/util/test-utils/seed.ts

```diff
@@ -183,25 +185,36 @@ async function generateOrg(admin: User, numUsers: number, numRosterEntries: numb
 
   // Insert observations per roster entry
   let observations: Observation[] = [];
-  for (let i = 0; i < numRosterEntries; i++) {
-    observations = observations.concat(
-      observationTestData(rosterEntries[i].edipi,
-        rosterEntries[i].unit.name,
-        reportSchemas[0],
-        moment('2020-01-01T08:00:00Z'),
-        moment('2020-01-07T08:00:00Z')),
-    );
-
-    // these observations are for the single-muster config
-    // hence the same timestamp for start and end
-    observations = observations.concat(
-      observationTestData(rosterEntries[i].edipi,
-        rosterEntries[i].unit.name,
-        reportSchemas[1],
-        moment('2020-01-02T10:00:00Z'),
-        moment('2020-01-02T10:00:00Z')),
-    );
-  }
+  const rosterEntriesByUnit = _.groupBy(rosterEntries, entry => entry.unit.name);
+  // for each unit...
+  _.forOwn(rosterEntriesByUnit, (value: Roster[]) => {
```

Comment:
`value` represents `rosters` so maybe there is a better name better describing what it is?